sql/moz-fx-data-shared-prod/telemetry_derived/ech_adoption_rate_v1/query.sql

```diff
@@ -5,13 +5,14 @@ WITH unioned AS (
     'http3_ech_outcome' AS metric,
     h.key AS label,
     v.key AS key,
-    v.value AS value,
+    COUNT(DISTINCT client_info.client_id) AS client_count,
+    SUM(v.value) AS handshakes,
     CASE
-      WHEN COUNT(DISTINCT client_info.client_id) >= 5000
+      WHEN SUM(v.value) >= 5000
+        OR COUNT(DISTINCT client_info.client_id) >= 5000
         THEN metadata.geo.country
       ELSE 'OTHER'
     END AS country_code,
-    COUNT(DISTINCT client_info.client_id) AS client_count
   FROM
     `moz-fx-data-shared-prod.firefox_desktop.metrics`,
     UNNEST(metrics.labeled_custom_distribution.http3_ech_outcome) AS h,
```

Comment:
If there are no http3 ourcomes, would the value be null or an empty array?

Reply:
It will be `null`, hence the filter below
`AND metrics.labeled_custom_distribution.http3_ech_outcome IS NOT NULL`